src/utils/paths.ts

```diff
@@ -1,20 +1,20 @@
-import { cwd } from 'node:process'
 import { join as joinDesm } from 'desm'
-import { resolve, join } from 'node:path'
+import { join, resolve } from 'node:path'
+import { cwd } from 'node:process'
 
 const workingDirectory = cwd()
 const packageRoot = joinDesm(import.meta.url, '..', '..')
 
-const resolvePath = (basePath, ...pathParts) => {
+const resolvePath = (basePath: string, ...pathParts: string[]): string => {
   try {
```

Comment:
Is this try/catch still necessary now that the `resolvePath` parameters are typed? `resolve` and `join` only throw when any of their arguments are not strings.

Reply:
Technically speaking, yes!